installer_scripts/cop_install_podnet_a.py

```diff
@@ -121,14 +121,14 @@ def build(win):
 
     # 1.3.2 Configure oob interface
     # sort ipaddresses
-    oob_ip = f'10.{config_data["pod_number"]}.0.254'
+    oob_ip = f'100.64.{config_data["pod_number"]}.254'
     configured, error = net.build(
         host='localhost',
         identifier=oob_iflname,
-        ips=[f'{oob_ip}/16'],
+        ips=[f'{oob_ip}/24'],
         mac=oob_mac,
         name='oob0',
-        routes=[{'to': '10.0.0.0/8', 'via': '10.0.0.1'}],
+        routes=[{'to': '100.64.0.0/16', 'via': '100.64.0.1'}],
```

Comment:
```suggestion
        routes=[{'to': '100.64.0.0/10', 'via': '100.64.0.1'}],
```
RFC 6598 allocation is from 100.64.0.0 to 100.127.255.255 which is a /10

https://datatracker.ietf.org/doc/html/rfc6598